tests/backpressure_test/Makefile

```diff
@@ -27,26 +27,24 @@ SHARED_FLAGS = -O2 -report -g -DSIM=1 -DDEBUG_PRINT_ENABLE=1 -DGENERATE_MCLK
 # XCC_XC_FLAGS, XCC_C_FLAGS, XCC_ASM_FLAGS, XCC_CPP_FLAGS
 # If the variable XCC_MAP_FLAGS is set it overrides the flags passed to
 # xcc for the final link (mapping) stage.
-XCC_FLAGS_SIM = $(SHARED_FLAGS) -DRECEIVE_DELAY_INCREMENT=0 -DSEND_DELAY_INCREMENT=10
 
-$(foreach db,8 16 24 32, \
-  $(foreach sr,768000 384000 192000,\
-  $(foreach chan,1 4,\
-    $(eval \
-      XCC_FLAGS_$(db)_$(sr)_$(chan)_5_5 = $(SHARED_FLAGS) -DDATA_BITS=$(db) -DSAMPLE_FREQUENCY=$(sr) -DNUM_I2S_LINES=$(chan) -DRECEIVE_DELAY_INCREMENT=5 -DSEND_DELAY_INCREMENT=5\
-      ) \
-    $(eval \
-      XCC_FLAGS_$(db)_$(sr)_$(chan)_10_0 = $(SHARED_FLAGS) -DDATA_BITS=$(db) -DSAMPLE_FREQUENCY=$(sr) -DNUM_I2S_LINES=$(chan) -DRECEIVE_DELAY_INCREMENT=10 -DSEND_DELAY_INCREMENT=0\
-      ) \
-    $(eval \
-      XCC_FLAGS_$(db)_$(sr)_$(chan)_0_10 = $(SHARED_FLAGS) -DDATA_BITS=$(db) -DSAMPLE_FREQUENCY=$(sr) -DNUM_I2S_LINES=$(chan) -DRECEIVE_DELAY_INCREMENT=0 -DSEND_DELAY_INCREMENT=10\
+SAMPLE_RATES ?= 768000 384000 192000
+CHANS ?= 1 4
+RX_TX_INCS ?= 5;5 10;0 0;10
+BITDEPTHS ?= 8 16 32
```

Comment:
Yes - thank you for reminding me to raise an issue. The 24b depth backpressure test fails because it takes longer to process a 24b payload than is accepted by the test - test probably wants refactoring to reflect this and acceptable backpressure values determined.